linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_clients.js

```diff
@@ -0,0 +1,105 @@
+var RouterClient = (function() {
+  var template;
+
+  function getMetricDefinition(routerName, clientName) {
+    return [
+      {
+        description: "Requests",
+        metricSuffix: "requests",
+        query: Query.clientQuery().withRouter(routerName).withClient(clientName).withMetric("requests").build()
```

Comment:
Yeah, I understand your thought process. Sounds like jQuery style could be a good compromise